package.json

```diff
@@ -8,6 +8,7 @@
     "dist",
     "lib",
     "lib/constants",
+    "lib/icons",
```

Comment:
We import directly from here, so I think it should also be present here?

Reply:
yeah, it seems you have to manually specify the nested paths here so I don't know how it was working. I remember I had to specify in daylight exactly ```  "files": [
    "dist",
    "dist/static",
    "dist/static/media"
  ],``` otherwise it would not output the media folder . But whatever this seems consistent